Microsoft.Toolkit.Uwp.UI.Controls.Input/RichSuggestBox/SuggestionChosenEventArgs.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Toolkit.Deferred;
+using Windows.UI.Text;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Provides data for the <see cref="RichSuggestBox.SuggestionChosen"/> event.
+    /// </summary>
+    public class SuggestionChosenEventArgs : DeferredEventArgs
+    {
+        /// <summary>
+        /// Gets the query used for this token.
+        /// </summary>
+        public string Query { get; internal set; }
+
+        /// <summary>
+        /// Gets the prefix character used for this token.
+        /// </summary>
+        public string Prefix { get; internal set; }
+
+        /// <summary>
+        /// Gets or sets the display text.
+        /// </summary>
+        public string Text { get; set; }
+
+        /// <summary>
+        /// Gets the suggestion item associated with this token.
+        /// </summary>
+        public object SelectedItem { get; internal set; }
+
+        /// <summary>
+        /// Gets token ID.
+        /// </summary>
+        public Guid Id { get; internal set; }
+
+        /// <summary>
+        /// Gets the formatting construct to override this token formatting.
```

Comment:
Not sure what this means as it's also read-only

Reply:
@michael-hawker The intention was to only allow the dev to set the formatting by changing the content of this property without replacing it. In retrospect, I think it's safe to let the dev use their own instance of `ITextCharacterFormat` if they want to. If they manually set the property to null, we can use the default token formatting instead. What do you think?

Reply:
Yeah that works, I guess I'm not too familiar with `ITextCharacterFormat` so maybe a better description could be `Gets the <see cref="ITextCharacterFormat"/> formatting description to override the default token formatting when displaying this token text.`?

Reply:
I have updated the summary to the following: `Gets or sets the <see cref="ITextCharacterFormat"/> object used to format the display text for this token.`. It's supposed to convey whatever the value of this property will be used to format the display text. The default token formatting should be the default value for this property. Let me know what you think.